index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="es">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mi CV</title>
+    <link rel="stylesheet" href="estilos.css"/>
+</head>
+
+<body>
+
+    <header>
+        <h1>Sergio Tomás Agudelo Rivera</h1>
+        <p>Estudiante de ingeniería mecánica de sexto semestre en la Universidad de la Sabana</p>  
+    </header>
+
+    <main id="contenedor">
+
+        <section class="item1">
+            <section id="content">
```

Comment:
Tener un section dentro de un section no es lo más adecuado, yo hubiera dejado solo un section y el otro como un div (puesto que su principal tarea es organizar)

```suggestion
        <section class="item1">
            <div id="content">
```